common/changes/@microsoft/api-documenter/updateApiModel_2019-06-08-04-52.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-documenter",
+      "comment": "Add support for emitting type parameters in the YamlDocumenter",
```

Comment:
@rbuckton  Did you consider updating `MarkdownDocumenter`?  If you don't have time, I'm okay with merging this as-is and someone else from the community will likely come along and finish that piece.

Reply:
I didn't look at `MarkdownEmitter` at all, but could consider it in a separate PR later if I have the time.